packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -334,119 +334,137 @@ export function GraphiQLInterface(props: GraphiQLInterfaceProps) {
     );
 
   return (
-    <div data-testid="graphiql-container" className="graphiql-container">
-      <div className="graphiql-sidebar">
-        <div className="graphiql-sidebar-section">
-          {pluginContext?.plugins.map(plugin => {
-            const isVisible = plugin === pluginContext.visiblePlugin;
-            const label = `${isVisible ? 'Hide' : 'Show'} ${plugin.title}`;
-            const Icon = plugin.icon;
-            return (
-              <Tooltip key={plugin.title} label={label}>
-                <UnStyledButton
-                  type="button"
-                  className={isVisible ? 'active' : ''}
-                  onClick={() => {
-                    if (isVisible) {
-                      pluginContext.setVisiblePlugin(null);
-                      pluginResize.setHiddenElement('first');
-                    } else {
-                      pluginContext.setVisiblePlugin(plugin);
-                      pluginResize.setHiddenElement(null);
-                    }
-                  }}
-                  aria-label={label}
-                >
-                  <Icon aria-hidden="true" />
-                </UnStyledButton>
-              </Tooltip>
-            );
-          })}
-        </div>
-        <div className="graphiql-sidebar-section">
-          <Tooltip label="Re-fetch GraphQL schema">
-            <UnStyledButton
-              type="button"
-              disabled={schemaContext.isFetching}
-              onClick={() => schemaContext.introspect()}
-              aria-label="Re-fetch GraphQL schema"
-            >
-              <ReloadIcon
-                className={schemaContext.isFetching ? 'graphiql-spin' : ''}
-                aria-hidden="true"
-              />
-            </UnStyledButton>
-          </Tooltip>
-          <Tooltip label="Open short keys dialog">
-            <UnStyledButton
-              type="button"
-              onClick={() => setShowDialog('short-keys')}
-              aria-label="Open short keys dialog"
-            >
-              <KeyboardShortcutIcon aria-hidden="true" />
-            </UnStyledButton>
-          </Tooltip>
-          <Tooltip label="Open settings dialog">
-            <UnStyledButton
-              type="button"
-              onClick={() => setShowDialog('settings')}
-              aria-label="Open settings dialog"
-            >
-              <SettingsIcon aria-hidden="true" />
-            </UnStyledButton>
-          </Tooltip>
-        </div>
-      </div>
-      <div className="graphiql-main">
-        <div
-          ref={pluginResize.firstRef}
-          style={{
-            // Make sure the container shrinks when containing long
-            // non-breaking texts
-            minWidth: '200px',
-          }}
-        >
-          <div className="graphiql-plugin">
-            {PluginContent ? <PluginContent /> : null}
+    <Tooltip.Provider>
```

Comment:
A lot of lines was changed due to wrapping app with `<Tooltip.Provider>`

Reply:
All good, reviewing with hidden whitespace makes this a breeze 😉